content/src/service/reporters/SQSDeploymentReporter.ts

```diff
@@ -2,27 +2,46 @@ import log4js from "log4js";
 import { EntityType } from "dcl-catalyst-commons";
 import { Entity } from "../Entity";
 import { DeploymentReporter } from "./DeploymentReporter";
-import { EthAddress } from "dcl-crypto";
+import { Authenticator, EthAddress } from "dcl-crypto";
 import SQS, { SendMessageRequest } from "aws-sdk/clients/sqs";
 import { AWSError, Credentials } from "aws-sdk";
+import { Environment, EnvironmentConfig } from "../../Environment";
+import { DeploymentEvent, MetaverseContentService } from '../Service';
 
 export class SQSDeploymentReporter implements DeploymentReporter {
 
     private static readonly LOGGER = log4js.getLogger('SQSDeploymentReporter');
 
     private sqsClient: SQS;
+    private queueURL: string;
 
     constructor(
-        accessKeyId: string,
-        secretAccessKey,
-        private readonly queueURL: string,
-        private readonly callback?: (error?:string, messageId?:string) => void) {
-
-        this.sqsClient = new SQS({
-            region: "us-east-1",
-            credentials: new Credentials(accessKeyId, secretAccessKey)})
+        env: Environment,
```

Comment:
Let's not take the `Environment` here, and just read everything on the factory. It's easier for mocking purposes. We can pass the different variables as we did before, but they would be of type `string | undefined` now.